packages/ui/src/components/cart/CartPopover.tsx

```diff
@@ -0,0 +1,519 @@
+                      width: 20,
+                      borderRadius: '99999px',
+                      ml: '$2',
+                    }}
+                  >
+                    <Text style="subtitle2">{items.length}</Text>
+                  </Flex>
+                )}
+                {!isCartEmpty && (
+                  <Text
+                    style="subtitle2"
+                    color="accent"
+                    css={{
+                      cursor: 'pointer',
+                      ml: 24,
+                      '&:hover': { color: '$accentSolidHover' },
+                    }}
+                    onClick={clear}
+                  >
+                    Clear All
+                  </Text>
+                )}
+                <Button
+                  size="none"
+                  color="ghost"
+                  css={{ color: '$neutralSolid', ml: 'auto' }}
+                  onClick={() => {
+                    setOpen(false)
+                  }}
+                >
+                  <FontAwesomeIcon icon={faClose} width="16" height="16" />
+                </Button>
+              </Flex>
+              {flaggedItems.length > 0 && (
+                <CartToast
+                  kind="warning"
+                  message={`${flaggedItems.length} ${flaggedItemsSubject} not tradeable on OpenSea`}
+                  link={
+                    <Text
+                      color="accent"
+                      style="subtitle2"
+                      css={{ ml: 'auto', mt: 3, cursor: 'pointer' }}
+                      onClick={(e) => {
+                        e.preventDefault()
+                        remove(
+                          flaggedItems.map(
+                            (item) => `${item.collection.id}:${item.token.id}`
+                          )
+                        )
+                      }}
+                    >
+                      Remove {flaggedItemsSubject}
+                    </Text>
+                  }
+                />
+              )}
+              {unavailableItems.length > 0 && (
+                <CartToast
+                  kind="error"
+                  message={`${unavailableItems.length} ${unavailableItemsSubject} no longer available`}
+                  link={
+                    <Text
+                      color="accent"
+                      style="subtitle2"
+                      css={{ ml: 'auto', mt: 3, cursor: 'pointer' }}
+                      onClick={(e) => {
+                        e.preventDefault()
+                        remove(
+                          unavailableItems.map(
+                            (item) => `${item.collection.id}:${item.token.id}`
+                          )
+                        )
+                      }}
+                    >
+                      Remove {unavailableItemsSubject}
+                    </Text>
+                  }
+                />
+              )}
+              {priceChangeItems.length > 0 && (
```

Comment:
This needs the singular/plural handling. Will be "1 prices updated" when priceChangeItems is equal to 1